.github/workflows/integrationtests.yaml

```
@@ -100,3 +100,16 @@ jobs:
 
       - name: Daps Integration Tests
         run: ./gradlew -p extensions/iam/daps test -DincludeTags="DapsIntegrationTest"
+
+  Micrometer-Integration-Tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+      - name: Download opentelemetry javaagent.
+        run: wget https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.10.1/opentelemetry-javaagent.jar
+      - name: Micrometer Integration Tests.
+        run: ./gradlew -p core/micrometer test -DincludeTags="MicrometerIntegrationTest"
```

```suggestion
        run: ./gradlew -p core/micrometer test -DincludeTags="OpenTelemetryIntegrationTest"
```
would probably make this more reusable (for other tests needed Otel agent)